pwa/src/pages/test.tsx

```diff
@@ -0,0 +1,23 @@
+import React, { useEffect, useState } from "react";
+import APIService from "../apiService/apiService";
+import APIContext from "../apiService/apiContext";
+
+const ExamplePage = (props: any) => {
```

Comment:
We're not using `props` in this component, and I would therefore leave it out.

Reply:
Also, we did not type our `ExamplePage` here. So typescript doesn't know what it is. I'd suggest updating it to:

```
ExamplePage: React.FC = () => { ... }
```

Reply:
Note: if you were to add props (which I wouldn't in this documentation), we usually make an interface and set it in the component declaration, like so:


```
interface ExamplePageProps {
  propName: string;
  nameProp: boolean;
  ...
}

const ExamplePage: React.FC<ExamplePageProps> = () => { ... }
```